src/content/reference/web-interface/_index.md

```diff
@@ -0,0 +1,31 @@
++++
+title = "Web Interface Reference"
+description = "Documentation on our web interface, a visual way to create and delete clusters, create key pairs and more."
```

Comment:
```suggestion
description = "Documentation on our web user interface, a visual way to manage clusters, create key pairs and more."
```

Reply:
I don't think this extra word here is really needed.